protocol/chainlib/rest.go

```diff
@@ -63,11 +64,20 @@ func (apip *RestChainParser) ParseMsg(url string, data []byte, connectionType st
 		return nil, err
 	}
 
+	// Extract default block parser
+	blockParser := serviceApi.BlockParsing
+
 	apiInterface := GetApiInterfaceFromServiceApi(serviceApi, connectionType)
 	if apiInterface == nil {
 		return nil, fmt.Errorf("could not find the interface %s in the service %s", connectionType, serviceApi.Name)
 	}
 
+	// Check if custom block parser exists in the api interface
+	// Use custom block parser only for URI calls
+	if apiInterface.GetOverwriteBlockParsing() != nil {
```

Comment:
Here as well no need to overwrite block parsing. this is only for tendermint. URI calls

Reply:
fixed: 524cb1833ba85f59885c6f6e829b7f9d6d08e451